### DISCUSSION TITLE
Trying to Get Game Time of Games

### DISCUSSION THREAD

**Discussion by bagelboy612** [cwendt94/espn-api#292] [ESPN API]
I am trying to use the getNFLGamesForPeriod method as described [here](http://espn-fantasy-football-api.s3-website.us-east-2.amazonaws.com/Client.html#getNFLGamesForPeriod). But I do not know what to run the method on? What should replace the X's in my sample here?

`import espn_api.football`
`games = XXXXXXXX.getNFLGamesForPeriod("20211202","20211208")`

**Comment by cwendt94** [comment]
That documentation is for another package, you can find all the information about this package in the [wiki](https://github.com/cwendt94/espn-api/wiki)
